keyboards/eyeohdesigns/babyv/readme.md

```diff
@@ -0,0 +1,61 @@
+# babyv
```

Comment:
This file should really follow the template <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/rules.mk>

Reply:
Fixed.

Reply:
I originally thought this was for the rules.mk, but saw you requested changes for the readme. Requested changes to the readme made.